File: app/src/main/java/com/appsbygu/kadai_weather_java/models/apis/Cities/Channel.java
package com.appsbygu.kadai_weather_java.models.apis.Cities;

import org.simpleframework.xml.Element;
import org.simpleframework.xml.Namespace;
import org.simpleframework.xml.Root;

import io.realm.RealmObject;


@Root(strict = false)
public class Channel extends RealmObject {

    @Element(required = false)
    private String author;

    @Element(required = false)
    private String title;

    @Namespace(reference="http://weather.livedoor.com/%5C/ns/rss/2.0")
    @Element(required = false)
    private Source source;

    @Element(required = false)
    private String lastBuildDate;

    public String getAuthor ()
    {
        return author;
    }

    public String getTitle ()
    {
        return title;
    }

    public Source getSource ()
    {
        return source;
    }

    public void setAuthor(String author) {
        this.author = author;
    }

    public void setTitle(String title) {
        this.title = title;
    }

    public void setSource(Source source) {
        this.source = source;
    }

    public String getLastBuildDate() {
        return lastBuildDate;
    }

    public void setLastBuildDate(String lastBuildDate) {
        this.lastBuildDate = lastBuildDate;
    }

}

File: app/src/main/java/com/appsbygu/kadai_weather_java/models/apis/Cities/Channel.kt
package com.appsbygu.kadai_weather_java.models.apis.Cities

import org.simpleframework.xml.Element
import org.simpleframework.xml.Namespace
import org.simpleframework.xml.Root

import io.realm.RealmObject


@Root(strict = false)
open class Channel : RealmObject() {

    @field:Element(name= "author", required = false)
    var author: String? = null

    @field:Element(name= "title", required = false)
    var title: String? = null

    @Namespace(reference = "http://weather.livedoor.com/%5C/ns/rss/2.0")
    @field:Element(name= "source", required = false)
    var source: Source? = null

    @field:Element(name= "lastBuildDate", required = false)
    var lastBuildDate: String? = null

}